code/game/objects/structures/tables_racks.dm

```diff
@@ -275,18 +275,20 @@
 			if(user.a_intent == INTENT_HARM)
 				if(user.grab_level > GRAB_AGGRESSIVE)
 					if (prob(15)) M.apply_effect(5, WEAKEN)
+					playsound(loc, 'sound/weapons/tablehit1.ogg', 25, 1, 7)
```

Comment:
Aren't we already playing this sound already in this block?

Reply:
Woops, my bad, fixed 😳